googletest/test/gtest_unittest.cc

```diff
@@ -4551,9 +4555,23 @@ TEST(ExpectTest, EXPECT_EQ_0) {
 TEST(ExpectTest, EXPECT_NE) {
   EXPECT_NE(6, 7);
 
+  constexpr auto lowest_int64 = ::std::numeric_limits<::std::int64_t>::lowest();
+  constexpr auto min_int64 = (::std::numeric_limits<::std::int64_t>::min)();
```

Comment:
Extra pair of parenthesis not needed here. Compare with the previous line.

Reply:
@kuzkry Are you sure the extra pair of parenthesis not needed here? When I remove them, VS2019 says:

> gtest_unittest.cc(4559,69): error C2220: the following warning is treated as an error
> gtest_unittest.cc(4559,69): warning C4003: not enough arguments for function-like macro invocation 'min'
> gtest_unittest.cc(4559,69): error C2589: '(': illegal token on right side of '::'
> gtest_unittest.cc(4559,69): error C2062: type 'unknown-type' unexpected
> gtest_unittest.cc(4559,69): error C2059: syntax error: ')'
> gtest_unittest.cc(4563,1): error C3536: 'min_int64': cannot be used before it is initialized

As it tries to call the `min(a,b)`macro from "minwindef.h".

Reply:
LOL, I didn't expect that 😄 I only verified it on my Linux machine. You're right

Reply:
@kuzkry No problem, thanks! Instead of the extra parentheses, I could have added `#define NOMINMAX`, or `#undef min`, but some people may not like those hashtags either 😸  

Thanks also for your approval  😃